orders.go

```diff
@@ -120,6 +120,41 @@ func (repository orderRepository) Create(order shopify.Order) (shopify.Order, er
 	return response.Order.ToShopify(), nil
 }
 
+func (repository orderRepository) Update(order shopify.Order) (shopify.Order, error) {
+
+	updateOrderDTO := BuildOrderDTO(order)
+
+	request := struct {
+		Order OrderDTO `json:"order"`
+	}{
+		Order: updateOrderDTO,
+	}
+
+	body, err := json.Marshal(request)
+
+	if err != nil {
+		return shopify.Order{}, err
+	}
+
+	url := repository.createURL(fmt.Sprintf("orders/%d.json", updateOrderDTO.ID))
+
+	respBody, _, err := repository.client.Put(url, body, nil)
```

Comment:
```suggestion
	responseBody, _, err := repository.client.Put(url, body, nil)
```
To keep it consistent with the create method